automated-comments/csharp/general/use_null_coalescing_operator_not_null_check.md

```diff
@@ -0,0 +1 @@
+Rather than explicitly checking for a value to equal `null` and then doing something with it, consider using the [null-coalescing operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator) to simplify your code.
```

Comment:
I would invert this per https://github.com/exercism/automated-mentoring-support/pull/61.

```suggestion
Use the [null-coalescing operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator) to simplify your code, rather than explicitly checking for a value to equal `null` and then doing something with it.
```